client/src/main/java/io/pravega/client/stream/impl/ReaderGroupImpl.java

```diff
@@ -376,6 +376,11 @@ private StreamCut computeEndStreamCut(Stream stream, Map<Segment, Long> endSegme
                       .thenApply(v -> completeCheckpointAndFetchStreamCut(checkpointId));
     }
 
+    @Override
```

Comment:
When ReaderGroup Configuration is updated or ReaderGroup is deleted, we need to make appropriate API calls. See this section in Design doc for details:
https://github.com/pravega/pravega/wiki/PDP-47:-Pravega-Streams:-Consumption-Based-Retention#updating-reader-group-configuration

Reply:
I've added the appropriate API calls to the controller at points mentioned in the PDP.